src/applications/pokemon_summary_screen/sprite.c

```diff
@@ -574,292 +639,282 @@ void PokemonSummaryScreen_DrawSheenSprites(PokemonSummaryScreen *summaryScreen)
         break;
     }
 
-    for (i = 29; i < 29 + summaryScreen->sheenMax; i++) {
-        if (CellActor_GetDrawFlag(summaryScreen->unk_41C[i]) == FALSE) {
+    for (i = PSS_SHEEN_SPRITES_START; i < PSS_SHEEN_SPRITES_START + summaryScreen->sheenMax; i++) {
+        if (CellActor_GetDrawFlag(summaryScreen->sprites[i]) == FALSE) {
             continue;
         }
 
-        CellActor_UpdateAnim(summaryScreen->unk_41C[i], FX32_ONE);
+        CellActor_UpdateAnim(summaryScreen->sprites[i], FX32_ONE);
 
-        if (CellActor_GetAnimFrame(summaryScreen->unk_41C[i]) == 6) {
-            CellActor_SetDrawFlag(summaryScreen->unk_41C[i], FALSE);
+        if (CellActor_GetAnimFrame(summaryScreen->sprites[i]) == 6) {
+            CellActor_SetDrawFlag(summaryScreen->sprites[i], FALSE);
         }
     }
 }
 
-void sub_0208F574(PokemonSummaryScreen *param0)
+void PokemonSummaryScreen_UpdateMiscMonDataSprites(PokemonSummaryScreen *summaryScreen)
 {
-    u32 v0;
-
-    for (v0 = 0; v0 < 6; v0++) {
-        if (param0->monData.markings & (1 << v0)) {
-            CellActor_SetAnim(param0->unk_41C[23 + v0], 1);
+    for (u32 marking = 0; marking < MAX_POKEMON_MARKINGS; marking++) {
+        if (summaryScreen->monData.markings & (1 << marking)) {
+            CellActor_SetAnim(summaryScreen->sprites[PSS_MARKING_SPRITES_START + marking], TRUE);
         } else {
-            CellActor_SetAnim(param0->unk_41C[23 + v0], 0);
+            CellActor_SetAnim(summaryScreen->sprites[PSS_MARKING_SPRITES_START + marking], FALSE);
         }
     }
 
-    if ((param0->monData.isShiny == 1) && (param0->monData.isEgg == 0)) {
-        SpriteActor_DrawSprite(param0->unk_41C[53], 1);
+    if (summaryScreen->monData.isShiny == TRUE && summaryScreen->monData.isEgg == FALSE) {
+        SpriteActor_DrawSprite(summaryScreen->sprites[PSS_SPRITE_SHINY_ICON], TRUE);
     } else {
-        SpriteActor_DrawSprite(param0->unk_41C[53], 0);
+        SpriteActor_DrawSprite(summaryScreen->sprites[PSS_SPRITE_SHINY_ICON], FALSE);
     }
 
-    if (param0->monData.pokerus == 2) {
-        SpriteActor_DrawSprite(param0->unk_41C[54], 1);
+    if (summaryScreen->monData.pokerus == PSS_POKERUS_CURED) {
+        SpriteActor_DrawSprite(summaryScreen->sprites[PSS_SPRITE_POKERUS_CURED_ICON], TRUE);
     } else {
-        SpriteActor_DrawSprite(param0->unk_41C[54], 0);
+        SpriteActor_DrawSprite(summaryScreen->sprites[PSS_SPRITE_POKERUS_CURED_ICON], FALSE);
     }
 }
 
-void sub_0208F600(PokemonSummaryScreen *summaryScreen)
+void PokemonSummaryScreen_ButtonPulseAnim(PokemonSummaryScreen *summaryScreen)
```

Comment:
**suggestion**: It's not clear to me what a `pulse` is. Is this the little animation that plays when you tap one of the buttons on the bottom screen?